src/components/CardLayout/CardLayout.js

```diff
@@ -35,7 +36,7 @@ function CardLayout({ children, columnWidthMin, rowHeight, ...props }) {
 CardLayout.propTypes = {
   children: PropTypes.node,
   columnWidthMin: PropTypes.number,
-  rowHeight: PropTypes.number,
+  rowHeight: PropTypes.oneOfType([PropTypes.string, PropTypes.number]),
```

Comment:
I think we can do this to validate `auto` and only `auto` :smile: 

```suggestion
  rowHeight: PropTypes.oneOfType([PropTypes.oneOf(['auto']), PropTypes.number]),
```

Reply:
Nice!